src/Schema/AbstractColumnSchema.php

```diff
@@ -270,8 +270,8 @@ protected function typecast(mixed $value): mixed
         }
 
         switch ($this->phpType) {
-            case Schema::PHP_TYPE_RESOURCE:
-            case Schema::PHP_TYPE_STRING:
+            case AbstractSchema::PHP_TYPE_RESOURCE:
```

Comment:
How about move this constants to SchemaInterface, and use SchemaInterface::PHP_TYPE_STRING etc